cmd/kueuectl/app/list/list_pods.go

```diff
@@ -60,20 +51,6 @@ the label selector or the field selector.`
 kueuectl list pods --for job/job-name`
 )
 
-var jobsWithPodLabelSelector = []JobWithPodLabelSelector{
-	&kueuejob.Job{},
-	&kueuejobset.JobSet{},
-	&kueuemxjob.JobControl{},
-	&kueuepaddlejob.JobControl{},
-	&kueuetfjob.JobControl{},
-	&kueuepytorchjob.JobControl{},
-	&kueuexgboostjob.JobControl{},
-	&kueuempijob.MPIJob{},
-	&pod.Pod{},
```

Comment:
uhm... I thought we remove the Pod already.

Does it mean we still have it enabled?

Reply:
Yes, because it allow this condition:

```
kueuectl list pods --for pod/{pod-name}
````

https://github.com/kubernetes-sigs/kueue/pull/2280#discussion_r1724535931.

Reply:
#2886